gui/default/assets/css/overrides.css

```diff
@@ -414,4 +419,12 @@ ul.three-columns li, ul.two-columns li {
     .btn:not(.panel-heading) + .btn:not(.panel-heading) {
         margin-bottom: 1rem;
     }
+
+    #folderFiles .modal-header {
+        min-height: 74px;
+    }
 }
+
+#folderFiles .modal-header {
+    min-height: 64px;
+}
```

Comment:
Can you give a rough explanation why we're adding these? Presumably the header would be as tall as the content needs, if that's the case, why do we need a min?